tests/test_event_handlers.py

```diff
@@ -209,13 +211,15 @@ def test_customer_subscription_deleted(
         event = self._create_event(FAKE_EVENT_CUSTOMER_SUBSCRIPTION_CREATED)
         event.invoke_webhook_handlers()
 
-        Subscription.objects.get(stripe_id=FAKE_SUBSCRIPTION["id"])
+        sub = Subscription.objects.get(stripe_id=FAKE_SUBSCRIPTION["id"])
+        self.assertEqual(sub.status, SubscriptionStatus.active)
 
         event = self._create_event(FAKE_EVENT_CUSTOMER_SUBSCRIPTION_DELETED)
         event.invoke_webhook_handlers()
 
-        with self.assertRaises(Subscription.DoesNotExist):
-            Subscription.objects.get(stripe_id=FAKE_SUBSCRIPTION["id"])
+        sub = Subscription.objects.get(stripe_id=FAKE_SUBSCRIPTION["id"])
+        # Check that Subscription is canceled and not deleted
+        self.assertEqual(sub.status, SubscriptionStatus.canceled)
```

Comment:
Should also check `sub.canceled_at = FAKE_SUBSCRIPTION["created"]`, shouldn't it?